### PR TITLE
Technomancer Shop, Disposal Remap and Maintenance Cyberpunkification 

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
some minor mapping edits, a small shop area for technomancers in their lobby, a remapping of disposals and some holoposters and decorations in and around maint. plus fixing that one tile in NT maint everyone hates
04-Aug-22 16-33-12
04-Aug-22 16-28-36
04-Aug-22 16-29-18

## Why It's Good For The Game
nicer mapping good

## Changelog
:cl:
add: more holoposters and decorations around maint
add: Shop area in technomancer lobby
tweak: Remapped Disposals
/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
fock

**Comment by Firefox13** [comment]
> some holoposters and decorations in and around maint

Wouldn't posters kill the whole grimy and forgotten mood that maints have?

**Comment by MLGTASTICa** [comment]
> > some holoposters and decorations in and around maint
> 
> Wouldn't posters kill the whole grimy and forgotten mood that maints have?

old posters

**Comment by ValoTheValo** [comment]
> > some holoposters and decorations in and around maint
> 
> Wouldn't posters kill the whole grimy and forgotten mood that maints have?

they are mainly around the entrances and high traffic areas

**Comment by Firefox13** [comment]
Fine I guess then.
Except integration tests didn't pass, you forgot a bracket on one of the tags it seems

File: maps/CEVEris/_CEV_Eris.dmm [hunk 1]
```diff
@@ -11662,11 +11665,8 @@
 /turf/simulated/floor/plating/under,
 /area/eris/maintenance/section3deck5starboard)
 "aCv" = (
-/obj/structure/disposalpipe/segment{
-	dir = 8;
-	icon_state = "pipe-c"
-	},
-/turf/simulated/floor/plating/under,
+/obj/structure/disposalpipe/segment,
+/turf/space,
```
Comment (Trilbyspaceclone) [inline maps/CEVEris/_CEV_Eris.dmm:11669]: Not sure this is intented to add a space tile were an underplating was

File: maps/CEVEris/_CEV_Eris.dmm [hunk 2]
```diff
@@ -104742,6 +105094,14 @@
 /mob/living/bot/cleanbot/roomba,
 /turf/simulated/floor/tiled/white,
 /area/eris/medical/medbay)
+"rkA" = (
+/obj/machinery/button/remote/blast_door{
+	id = TechnomancerShopLockdown;
```
Comment (Trilbyspaceclone) [inline maps/CEVEris/_CEV_Eris.dmm:105099]:
```suggestion
	id =  "TechnomancerShopLockdown";
```

Comment (ValoTheValo) [reply maps/CEVEris/_CEV_Eris.dmm]: for some reason this aint letting me commit this

Comment (ValoTheValo) [reply maps/CEVEris/_CEV_Eris.dmm]: manually did it

File: maps/CEVEris/_CEV_Eris.dmm (hunk 1 above)
Comment (ValoTheValo) [reply maps/CEVEris/_CEV_Eris.dmm:11669]: fixed

**Comment by ValoTheValo** [comment]
should be fully working now :clueless:

**Comment by ValoTheValo** [comment]
okay now its actually fixed

**Pull request merged**